onnxruntime/test/python/transformers/test_gqa_cpu.py

```diff
@@ -1591,12 +1702,26 @@ def parity_check_gqa_prompt_no_buff(
 
     head_sink = get_custom_head_sink(config.num_heads, torch_type=torch_type) if config.has_head_sink else None
 
+    output_qk = (
+        torch.zeros(
+            config.batch_size,
+            config.num_heads,
+            config.kv_sequence_length,
```

Comment:
This shape does not looks right. It shall be like (batch_size, num_head, q_sequence_length, total_sequence_length).